src/components/elementView/powerView/bladePowerView.tsx

```diff
@@ -94,8 +91,6 @@ export class BladePowerView extends React.PureComponent<
   componentDidUpdate() {
     if (this.props.shouldUpdate && this.shouldShowPower()) {
       this.getPowerStatus();
-    } else {
-      this.setState({ statusLoaded: true });
```

Comment:
this was causing a delay between when the status was supposedly loaded and when the value was actually returned from the endpoint